internal/host/transition_test.go

```diff
@@ -3148,6 +3149,7 @@ var _ = Describe("Refresh Host", func() {
 					BelongsToMachineCidr:        {status: ValidationSuccess, messagePattern: "Host belongs to machine network CIDR"},
 					IsNTPSynced:                 {status: ValidationSuccess, messagePattern: "Host NTP is synced"},
 					AreContainerImagesAvailable: {status: ValidationSuccess, messagePattern: "All required container images were pulled and are available"},
+					AreOcsRequirementsSatisfied: {status: ValidationFailure, messagePattern: "OCS unsupported Host Role for Compact Mode."},
 				}),
```

Comment:
@pkliczewski  @jakub-dzon  , should we cover more OCS + CNV test scenarios here?

Reply:
Would be great if we could add OCS + CNV but it can be done in separate PR

Reply:
I agreed with @pkliczewski. We need this fix ASAP for building a new release - let's do it in separate PR.